src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,79 +24,94 @@ redirects:
+</Callout>
+
+### Tell us where to send your alerts [#destination-intro]
+
+After you've created a workflow that connects your team with the information they need,  you need to configure your destinations. Destinations are where we send notifications about your New Relic data. A destination is a unique identifier for a third-party system that you use. Destination settings contain the connection details to integrate with third-party systems and can be used across a variety of tools in New Relic.
+
+To learn how to set up destinations and configure message templates, [explore our notification documentation](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/destinations/).
+
+## Navigate the alerts and applied intelligence UI [#get-started-with-alerts]
 
 <img
-  title="The alerts and applied intelligence Issue Feed"
-  alt="Screenshot of the Alerts and AI Issue Feed"
-  src={accountsIssueFeedforOverviewPage}
+  title="Alerts and AI homepage"
+  alt="A screenshot outlining the alerts and ai UI"
+  src={alertsAlertsandAiOverviewPage}
 />
 
-<figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI: You can use the issue feed to review all your issues.**
-</figcaption>
-
-In the New Relic UI, Alerts & AI is divided into several sections, each unlocking different features. Check out the section explanations below to learn how these features can meet your needs.
+In the New Relic UI, alerts and applied intelligence is divided into several sections, each unlocking different features. Check out the section explanations below to learn how these features can meet your needs.
 
-## **Analyze** your issues, incidents, and anomalies [#analyze]
+### Analyze your issues, incidents, and anomalies [#analyze]
 
 Use the analyze features to scan for critical problems affecting your systems, via alerts violations, issues, incidents, and anomalies.
 
 * **[Overview](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page)**: Find your most recent open, closed, and muted violations.
 * **Issues & activity**: Review your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
 
-## **Detect** problems in your system [#detect]
+### Detect problems in your system [#detect]
 
 Define conditions in your system you want to be notified about with alert policies.
 
 * **[Alert policies](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/)**: Create new policies to group related conditions together. Also, lists the policies where your conditions are created and organized.
```

Comment:
Policies are important too, and should get a mention, but conditions are vital. They should get top shelf, here, since they are the configuration items that actually do the work.

Suggestion: for the first line, change to

Define problems in your system you want to be notified about with alert conditions.

Reply:
done :)